### PR TITLE
[OSDEV-525] - Label Latitude and Longitude on the Page

### PR DESCRIPTION
**Pull request by VadimKovalenkoSNF**
[OSDEV-525](https://opensupplyhub.atlassian.net/browse/OSDEV-525), add latitude and longitude labels, minor UI changes.

On facility page
- Add 'Longitude' and 'Latitude' labels.

On facility page right sidebar (click '<number> more entries' under the Latitude / Longitude)
- Add 'Latitude Longitude' small header.

For both components replace 'GPS' header to 'COORDINATES' and applied correct order of Latitude and Longitude (Latitude should go first).
Add unit tests.

### CODE REVIEW

File: src/app/src/components/TitledDrawer.jsx [hunk 1]
```diff
@@ -50,7 +50,11 @@ const TitledDrawer = ({
             </div>
             <Typography className={classes.drawerTitle}>{title}</Typography>
             <Typography className={classes.drawerSubtitle}>
-                {subtitle}
+                <>
+                    {subtitle}
+                    <br />
+                    Latitude Longitude
```
Comment (vladsha-dev) [inline src/app/src/components/TitledDrawer.jsx:56]: 1. I think this text should be passed along with the `subtitle` prop. According to the UX mockups, it seems that the comma is missing between the words. -> ` Latitude, Longitude`.
2. Please don't forget to verify other places where the TitledDrawer component is used.
**Issue:**
Screenshot 2024-02-16 at 09 54 09

File: src/app/src/components/FacilityDetailsItem.jsx [hunk 2]
```diff
@@ -52,7 +53,7 @@ const FacilityDetailsItem = ({
                 <Typography className={classes.label}>{label}</Typography>
             </div>
             <FacilityDetailsDetail
-                primary={primary}
+                locationLabeled={locationLabeled}
```
Comment (vladsha-dev) [inline src/app/src/components/FacilityDetailsItem.jsx:56]: This change caused this bug:
Screenshot 2024-02-16 at 09 48 53

**Review by vladsha-dev — CHANGES_REQUESTED** [review]
I left the comments.

**Comment by vladsha-dev** [comment]
But let's leave this PR for now and focus on merging functionality since most of it is already in the main branch.

**Comment by roman-stolar** [comment]
In general LGTM, will approve when bug and merge conflicts will be fixed.

File: src/app/src/components/FacilityDetailsItem.jsx (hunk 2 above)
Comment (VadimKovalenkoSNF) [reply src/app/src/components/FacilityDetailsItem.jsx:57]: @vladsha-dev All comments have been fixed, pls re-check.

File: src/app/src/components/TitledDrawer.jsx (hunk 1 above)
Comment (VadimKovalenkoSNF) [reply src/app/src/components/TitledDrawer.jsx]: Fixed

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by vladsha-dev — CHANGES_REQUESTED** [review]
I am waiting for the fix of the Bug related to rendering the first item in the drawer.
It was discussed on the call with Vadim

**Comment by VadimKovalenkoSNF** [comment]
> I am waiting for the fix of the Bug related to rendering the first item in the drawer.
It was discussed on the call with Vadim

Done. FYI, the error with keys on opening the sidebar happens on main branch, this can be considered as tech debt.

**Review by roman-stolar — APPROVED** [review]
LGTM

**Comment by roman-stolar** [comment]
Please update your branch with latest “main”

**Pull request merged**